### PR TITLE
Add more signal tests.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -16,6 +17,14 @@ add_custom_target(
   WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
 )
 
+add_executable(
+  crash
+  crash.cc
+  )
+
+# add_dependencies(check crash)
```
Comment (kasperl) [inline tests/CMakeLists.txt:25]: In our out?

File: tests/pipe_crash_test.toit [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      pipe.PIPE_CREATED  // stdin
+      pipe.PIPE_CREATED  // stdiout
+      pipe.PIPE_CREATED  // stderr
+      crash_exe
+      [crash_exe] + signal_arg
+
+    pid := pipes[3]
+    pipe.wait_for pid
+
+
```
Comment (kasperl) [inline tests/pipe_crash_test.toit:30]: Drop one newline.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tests/pipe_crash_test.toit [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  if platform == "FreeRTOS": return
+
+  crash_exe := args[0]
+
+  run_crash := : | signal/int? |
+    signal_arg := signal ? ["$signal"] : []
+    pipes := pipe.fork
+      true  // use_path
+      pipe.PIPE_CREATED  // stdin
+      pipe.PIPE_CREATED  // stdiout
```
Comment (erikcorry) [inline tests/pipe_crash_test.toit:22]:
```suggestion
      pipe.PIPE_CREATED  // stdout
```

File: tests/pipe_crash_test.toit [hunk 4]
```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2019 Toitware ApS.
```
Comment (erikcorry) [inline tests/pipe_crash_test.toit:1]:
```suggestion
// Copyright (C) 2023 Toitware ApS.
```

**Review by erikcorry — APPROVED** [review]

File: tests/pipe_crash_test.toit (hunk 3 above)
Comment (floitsch) [reply tests/pipe_crash_test.toit]: done.

**Pull request merged**